app/main.py

```diff
@@ -2,0 +1,10 @@
+def copy_file(command: str) -> None:
+    cmd, origin, dupe = command.split()
+
+    if cmd != "cp" or origin == dupe:
```

Comment:
Code Efficiency: You correctly check whether the command is 'cp' and whether the origin and dupe are the same. However, you might want to give the user a feedback when these conditions are not met.